client/src/App.tsx

```diff
@@ -22,13 +22,13 @@ function App(): JSX.Element {
             <SocketProvider>
               <NavBar />
               <Switch>
-                <Route exact path="/login" component={Login} />
-                <Route exact path="/signup" component={Signup} />
-                <ProtectedRoute exact path="/dashboard" component={Dashboard} />
-                <ProtectedRoute exact path="/my-jobs" component={Dashboard} />
-                <ProtectedRoute exact path="/messages" component={Dashboard} />
-                <ProtectedRoute exact path="/my-sitters" component={Dashboard} />
-                <ProtectedRoute exact path="/edit-profile" component={EditMenu} />
+                <ProtectedRoute exact path="/login" component={Login} />
+                <ProtectedRoute exact path="/signup" component={Signup} />
+                <Route exact path="/dashboard" component={Dashboard} />
```

Comment:
We want to have these route as protected routes and the login/signup pages as unprotected routes :)